functions/util/github.ts

```diff
@@ -0,0 +1,37 @@
+const request = require('request');
+
+/** Function that sets a Github commit status */
+export function setGithubStatus(commitSHA: string,
+                                result: boolean,
+                                name: string,
+                                description: string,
+                                url: string,
+                                repoSlug: string,
+                                token: string) {
```

Comment:
As mentioned above, I would prefer the syntax / API we currently have on `tools/gulp/util/github.ts`